### ISSUE TITLE
GGMax - feature request - An option to restart the level on respawn

### ISSUE THREAD

**Issue by DjPoke** [TheGameCreators/GameGuruRepo#4446]
Hi,

If possible, it could be usefull if there were an option to reset level's data on respawn.
(Reset all enemies position)

**Comment by Tum1370** [comment]
This is the same as what we had and basically the same as what i ask here https://github.com/TheGameCreators/GameGuruRepo/issues/4409

This is a bug really, as it was taken away when they updated the system to work how it does now.
We need an option to be able to reset a whole level (If we want to).
And also we obviously need the method that is currently in place as well.

**Comment by Necrym59** [comment]
Thats basically starting a new game??

**Comment by DjPoke** [comment]
No. Only for level's data at the time the level is restarted. The problem i have in my game is that when enemies kill me, and when i respawn at start, they have moved to the respawn point. Note my game is not multiplayer, but solo.
